app/employee-app/backend/service.bal

```diff
@@ -95,4 +95,50 @@ service http:InterceptableService / on new http:Listener(9090) {
         }
         return employees;
     }
+
+    # Get basic information of employees.
```

Comment:
Incorrect documentation.

Reply:
Holding this until we finalize the requirement

Reply:
Removed after discussion.